apps/web/src/client/app.js

```diff
@@ -1,6 +1,8 @@
 // Import default components for all pages
 //* This will be updated based on a router -> loader flow.
 
+// eslint-disable-next-line no-unused-vars
+import regeneratorRuntime from 'regenerator-runtime';
```

Comment:
Why are we importing this? Doesn't look like we are using it?

Reply:
there is something going on with Babel that requires it - we are looking into it